docs/book/introduction.md

```diff
@@ -16,15 +16,135 @@ description: Welcome to ZenML!
+zenml stack set aws
+python run.py  # Now your ML workflow runs in AWS
+```
+
+**Extensibility:** ZenML is open-source and highly extensible, so you benefit 
+from industry-standard best-practices built into the framework while also being 
+able to customize it to fit your specific needs.
+
+{% endtab %}
+
+{% endtabs %}
 
 ## Directions
 
-If you liked what you have seen so far and you are eager to get started, here is a collection of pages you can take a look at next:
+Ready to harness the power of ZenML for your machine learning projects? Here is a collection of pages you can take a look at next:
 
 <table data-view="cards"><thead><tr><th></th><th></th><th></th><th data-hidden data-card-target data-type="content-ref"></th></tr></thead><tbody><tr><td><span data-gb-custom-inline data-tag="emoji" data-code="1f4bf">💿</span> <strong>Installation</strong></td><td>Install ZenML to get started.</td><td></td><td><a href="getting-started/installation.md">installation.md</a></td></tr><tr><td><span data-gb-custom-inline data-tag="emoji" data-code="1f9f1">🧱</span> <strong>Core Concepts</strong></td><td>Understand the core concepts behind ZenML.</td><td></td><td><a href="getting-started/core-concepts.md">core-concepts.md</a></td></tr><tr><td><span data-gb-custom-inline data-tag="emoji" data-code="1f423">🐣</span> <strong>Starter Guide</strong></td><td>Get started quickly with a simple setup.</td><td></td><td></td></tr><tr><td><span data-gb-custom-inline data-tag="emoji" data-code="1f414">🐔</span> <strong>Advanced Guide</strong></td><td>Hone your MLOps skills with an advanced guide.</td><td></td><td></td></tr><tr><td><span data-gb-custom-inline data-tag="emoji" data-code="1f4cb">📋</span> <strong>Component Guide</strong></td><td>Browse through the already-implemented integrations for ZenML.</td><td></td><td><a href="user-guide/component-galery/">component-galery</a></td></tr><tr><td><span data-gb-custom-inline data-tag="emoji" data-code="1f3d7">🏗</span> <strong>Set up your MLOps Platform</strong></td><td>Establish your production-ready infrastructure.</td><td></td><td><a href="platform-guide/set-up-your-mlops-platform/">set-up-your-mlops-platform</a></td></tr><tr><td><span data-gb-custom-inline data-tag="emoji" data-code="1f9d1-1f692">🧑🚒</span> <strong>Manage the Deployed Services</strong></td><td>Manage your production-ready infratstructure.</td><td></td><td><a href="platform-guide/manage-the-deployed-services/">manage-the-deployed-services</a></td></tr><tr><td><span data-gb-custom-inline data-tag="emoji" data-code="1f9d1-1f3eb">🧑🏫</span> <strong>Examples</strong></td><td>Take a peek at how ZenML works in concrete examples.</td><td></td><td><a href="learning/examples/">examples</a></td></tr><tr><td><span data-gb-custom-inline data-tag="emoji" data-code="1f64b">🙋</span> <strong>FAQ</strong></td><td>Find the answers to the most frequently asked questions.</td><td></td><td><a href="learning/faq.md">faq.md</a></td></tr></tbody></table>
```

Comment:
Personally, I feel like these are too many directions. If we have 9 i would some put them in the tabs, so 3 for each persona (ideally they would be linked to the values we provide, or a starting guide for them).

Alternatively (or additionally), give only 3 directions: User guide, Platform Guide, and Core Concepts for all personas

Reply:
With the current page, I agree with @htahir1 on this one. 9 directions might be a bit too much. However, considering that this is the introduction page, I would propose a different set of three pages: Quickstart, Core Concepts, Installation(or Starter Guide) in any order.

Reply:
Actually I like the current layout. I think having those tiles looks quite clean and is not overwhelming either. And it's a great first docs page where you can find quick links to what you need at a glance.

Adding persona-specific next steps could be interesting but I'm not 100% sure if you can split it cleanly and it could also hide relevant links below tabs that the user might not click.

Reply:
@bcdurak oh yeah the quickstart is missing from the tiles right now, good point. I just added it 👍